openedx2zim/scraper.py

```diff
@@ -433,13 +620,20 @@ def run(self):
         )
         logger.debug("Checking for missing binaries")
         check_missing_binary(self.no_zim)
+        logger.info("Checking S3 cache credentials")
+        if self.s3_url_with_credentials and not self.s3_credentials_ok():
+            raise ValueError("Unable to connect to Optimization Cache. Check its URL.")
+        if self.s3_storage:
+            logger.info(
+                f"Using cache: {self.s3_storage.url.netloc} with bucket: {self.s3_storage.bucket_name}"
+            )
         logger.debug("Testing credentials")
```

Comment:
This should be an INFO (you're testing values provided by user) and should be rephrased

Reply:
Changed that